### DISCUSSION TITLE
Need help to validate a YAML file using a schema

### DISCUSSION THREAD

**Discussion by bluelephant825** [redhat-developer/vscode-yaml#821] [Q&A] [answered]
I created a file named .pls.yml with the code below in my ~ home directory on Mac. I would like to validate the file with the schema provided by the url below. I installed the YAML extension for VS Code but nothing is happening. Can someone please explain what I'm missing. Thank you.

```
# yaml-language-server: $schema=https://raw.githubusercontent.com/dhruvkb/pls/main/src/pls/data/schema/pls_config.yml

nerd_icons:
  pdf: "\uf724" # nf-mdi-file_pdf
```

P.S. I've been trying to follow the instructions on [this page](https://dhruvkb.github.io/pls/reference/).

**Comment by gorkem** [comment, marked as answer]
The small example you have posted seems to work for me. Could there be an issue with the network connectivity? As vscode-yaml needs to be able to download the schema. Perhaps a proxy?

**Reply by bluelephant825** [reply]
@gorkem My fault! I hadn't understood how to use the YAML validation. If I make an error in the YAML file, it gets reported under Problems in VSC. Therefore, I assume things are working properly. 

<img width="1002" alt="Screenshot 2022-10-04 at 18 34 31" src="https://user-images.githubusercontent.com/4818315/193875980-18e5a7b5-e601-4ffd-bd50-39bac9f4f29d.png">


Sorry for my misunderstanding: I wasn't seeing anything happen because my YAML file didn't contain any errors!

**Reply by gorkem** [reply]
That does not still sound right. It should also show a red mark on the side for that line. But it looks like vscode-yaml is reporting the error as expected.
